CHANGELOG.md

```diff
@@ -19,6 +19,10 @@
 * `Profile` has increased margin between the image and text.
 * `Dialog` bottom padding has increased by 8px.
 
+## Bug Fixes
```

Comment:
`2.4.0` is the current release, I'd target this under `2.4.1` as it is a bug fix.